### ISSUE TITLE
about "#include dds_cdrstream_write.part.c and #include dds_cdrstream_keys.part.c"

### ISSUE THREAD

**Issue by zzzzfight** [eclipse-cyclonedds/cyclonedds#1855]
sorry, I just think we should change the  "dds_cdrstream_write.part.c" to "dds_cdrstream_write.part.h" and "dds_cdrstream_keys.part.c" to "dds_cdrstream_keys.part.h"

**Comment by eboasson** [comment]
That's actually a good point. I mean, I don't really care, but CMake does ... if we change them to `.h` I can add them to list of sources in CMake. @dpotman what do you think?

**Comment by dpotman** [comment]
Yes, I'm fine with this. There might be other ways to express the dependency on these files in CMake, but this is a pragmatic way of fixing this.
